packages/frontend/amp/components/Expandable.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import { pillarPalette } from '@frontend/lib/pillars';
+import InfoIcon from '@guardian/pasteup/icons/info.svg';
+import PlusIcon from '@guardian/pasteup/icons/plus.svg';
+
+import { body, textSans, headline } from '@guardian/pasteup/typography';
+import { palette } from '@guardian/pasteup/palette';
+
+// TODO - check if we need to use a palette colour for background (neutral 93 is
+// closest) or incorporate this one into the palette?
+const wrapper = (pillar: Pillar) => css`
+    background: #f1f1f1;
+    position: relative;
+    padding: 0 5px 6px;
+    margin: 16px 0 36px;
+
+    border-top: 13px solid ${palette.neutral[7]};
+    border-image: repeating-linear-gradient(
+            to bottom,
+            #dcdcdc,
+            #dcdcdc 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+
+    a {
```

Comment:
Adding styling for `a`, `p`, `b` etc feels wrong here - I should grab this from elsewhere as it's general styling for text markup.

Reply:
yeah I agree. So I am about to add a commit to export the styling from the `Text` element component and use that. But really we probably need to think about this. My own view is that it should live in Guui but I know at the moment that has been purely for actual React components.